apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -340,3 +340,11 @@ function load_coming_soon() {
 	}
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_coming_soon' );
+
+/**
+ * What's New section of the Tools menu
+ */
+function load_whats_new() {
```

Comment:
We should probably restrict this based on when it needs to be loaded. In this case, we probably want to make sure that it is only available when Gutenberg is loaded.